doc/compe.txt

```diff
@@ -298,12 +298,12 @@ DEVELOPMENT                                                          *compe-dev*
 ==============================================================================
 EXAMPLE SOURCE                                             *compe-dev-ex-source*
 
-You can see example on *https://github.com/kristijanhusak/vim-dadbod-completion*
```

Comment:
@elianiva `**` will always create a new tag, so this link was showing up in `:help` completion on the command line. Just to let you know :)

Reply:
thanks!